dev/benchmarks/multiple_flutters/android/app/build.gradle

```diff
@@ -37,7 +37,7 @@ android {
     defaultConfig {
         applicationId "dev.flutter.multipleflutters"
         minSdk = 24
-        targetSdk = 35
+        targetSdk = 36
```

Comment:
I would suggest modifying target sdk in a different pr. Target sdk changes are more likely to cause behavior changes in these benchmarks.